### PR TITLE
Resolves #12: Reviews and Ratings + Dishes and Chefs Databases

### LINKED ISSUE #12: [Order Process] Reviews and Ratings Records and Averages
## User Story
As a **User**, I would like **to know the average ratings of Dishes and Chefs and read written reviews left by other customers** so that **I can decide whether to continue ordering a certain Dish from a certain Chef or not**.

## Detailed Description
Every dish will have its own ratings and written reviews. The rating of a dish is the average star rating of the valid ratings left for that dish. A chef's rating and reviews will consist of the average of all ratings and reviews for any of their dishes. The ratings and reviews will be stored in a database indexed by chef and dish.

## Acceptance Criteria
- [x] Given **Developer runs a script**, then **database is accessible and app can pull from it**
- [x] Given **User starts Docker container**, then **Database is up and running locally**

## Tasks
- [x] Create Database for reviews
- [x] Create Database for Chefs
- [x] Create Database for Dishes

### PR DESCRIPTION
**Pull request by brendon-ng**
Closes #12: Includes backend for running database locally for app testing on development device. Also includes some updates to app structure to be finished in issue#6.

### CODE REVIEW

File: app/components/NavBarComponent.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+class NavBarComponent extends Component{
+    render(){
+        //console.log("NavBar Props");
+        //console.log(this.props);
+        return(
+            <Header
+                containerStyle={{backgroundColor: colors.primary}}
+                leftComponent={<LeftElement navigation={this.props.navigation}/>}//{icon:'search', color:'white', onPress: ()=> {console.log("Search Pressed")}}}
+                centerComponent={<CenterElement/>} //{ text: 'POTLUCK', style:{ color: '#fff', fontFamily:'Avenir', fontWeight:'bold', fontSize:20} }}
+                rightComponent={<RightElement/>}//{ icon: 'map', color: '#fff', onPress: ()=> {console.log("Map Pressed")}}}
```
Comment (jcgubbi) [inline app/components/NavBarComponent.js]: Can you delete this comments if they aren't relevant anymore to clean up code

File: app/screens/DishPage.js [hunk 2]
```diff
@@ -24,12 +25,18 @@ const CarouselCardItem = ({ item, index }) => {
     )
 }
 function DishPage(props) {
+    //console.log(props);
+    //props.push(props.route.params)
```
Comment (jcgubbi) [inline app/screens/DishPage.js]: Same issue as above. If this isn't relevant plz delete.

File: app/screens/FeaturedMenuScreen.js [hunk 3]
```diff
@@ -1,12 +1,169 @@
...
+        this.getDishes().then(function(results) {
+            if(results!=null){
+                //this.setState({dishes: results});
+                //console.log(results);
+                setDishes(true);
+                //console.log(this.state.test); 
+            }
+        }, ()=>{console.log("Then Error")})
+        .catch((err) => {console.log("Constructor Err: ", err)});
+    */
```
Comment (jcgubbi) [inline app/screens/FeaturedMenuScreen.js]: delete

**Review by jcgubbi — CHANGES_REQUESTED** [review]
Delete commented out code where relevant

**Review by shikham-8 — APPROVED** [review]
looks good!

**Review by jcgubbi — APPROVED** [review]

**Pull request merged**